UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
Ah, I don't, but that's probably because you did a Ctrl-I to re-indent your code and now have all the empty lines "indented" (i.e. your empty lines actually contain spaces)

You can check a CheckBox in Xcode > Preferences > Text Editing to prevent Xcode from adding those every time.